lib/ssoe/service.rb

```diff
@@ -20,6 +20,10 @@ class Service < Common::Client::Base
     ].freeze
 
     def get_traits(credential_method:, credential_id:, user:, address:)
+      if Rails.env.development?
+        Rails.logger.info('[SSOe::Service::get_traits] Mocked response in development')
+        return { success: true, icn: 'mocked-icn-123456' }
+      end
```

Comment:
We should mock this with betamocks. You can put the config in `config/betamocks/services_config.yml`. A good example is the [MHV UserAccount Creation mock](https://github.com/department-of-veterans-affairs/vets-api/blob/c0dd809dc50199f19ed55058e312d8e7d1317ec9/config/betamocks/services_config.yml#L186). The `file_path` is the path to the recorded response in [vets-api-mockdata](https://github.com/department-of-veterans-affairs/vets-api-mockdata/blob/master/mhv/account_creation/create_account/default.yml). It follows the same style as a VCR cassette